app/src/main/java/at/roteskreuz/stopcorona/model/managers/DatabaseCleanupManager.kt

```diff
@@ -65,6 +53,8 @@ class DatabaseCleanupManagerImpl(
             }
 
             val nowAsRollingStartIntervalNumber = ZonedDateTime.now()
```

Comment:
this is not "now" this is a day in the past, most likely 14 days in the past

Reply:
updated